app/helpers/errors_helper.rb

```diff
@@ -1,12 +1,18 @@
 module ErrorsHelper
-  def errors_for(errors, attribute)
+  def errors_for(errors, attribute, full_message: true)
```

Comment:
Maybe name this parameter to be clearer that it's expecting a boolean (e.g. `use_full_message`)? Or maybe even clearer, have two public methods that just have the `errors` and `attribute` parameters (one for using `message` and one for using `full_message`), and a private method that is this one with the boolean parameter (if you can have private methods in a module…?).